.fastlane/Fastfile

```diff
@@ -4,14 +4,74 @@ default_platform :ios
 
 platform :ios do
   before_all do
-    xcversion(version: "9.2")
+    setup_circle_ci
+    xcversion(version: ENV["XCODE_VERSION"])
+  end
+
+  ### MATCH
+
+  desc "Fastlane Match"
+  lane :match_all do
```

Comment:
This syncs up our certificates and provisioning profiles for Drip and Kickstarter.